generators/python/src/fern_python/generators/core_utilities/core_utilities.py

```diff
@@ -119,6 +129,12 @@ def get_serialize_datetime(self) -> AST.Reference:
             ),
         )
 
+    def get_fern_enum(self) -> AST.ClassReference:
+        return AST.ClassReference(
+            qualified_name_excluding_import=(),
+            import_=AST.ReferenceImport(module=AST.Module.local(*self._module_path, "enum"), named_import="StrEnum"),
```

Comment:
one more edge case:

Can we have this show up as:
```
import ..core.enum

class Foo(enum.StrEnum):
```

I want to avoid a conflict with a class named StrEnum